src/shared/api/urls.ts

```diff
@@ -324,7 +324,7 @@ export function getDarwinUrl(sampleIds: string[], caseId: string) {
 
 export function getStudyDownloadListUrl() {
     return buildCBioPortalAPIUrl(
-        'proxy/download.cbioportal.org/study_list.json'
+        'proxy/cbioportal-datahub.s3.amazonaws.com/study_list.json'
```

Comment:
don't need the proxy anymore since S3 is on https, so can directly use `https://cbioportal-datahub.s3.amazonaws.com/study_list.json`. Before we were using the proxy to be able to pull from http on a https domain